mmcv/runner/hooks/evaluation.py

```diff
@@ -99,11 +104,14 @@ def __init__(self,
             self.best_ckpt_path = None
```

Comment:
Is it possible to make greater_keys and less_keys configurable through the initialization arguments. Thus downstream tasks do not need to inherit a new module and modify greater_keys/less_keys anymore.

Reply:
Yes, it is possible. Even so, I think it's better to inherit a new EvalHook in downstream repos, to override the default settings like test_fn and greater/less keys. Or they have to be written in the configs, which seems not user-friendly.